docs/Using-Fleet/REST-API.md

```diff
@@ -5119,38 +5119,36 @@ Returns the query specified by ID.
     "name": "centos_hosts",
     "description": "",
     "query": "select 1 from os_version where platform = \"centos\";",
+    "team_id": null,
+    "interval": 3600,
+    "platform": "",
+    "min_osquery_version": "",
+    "automations_enabled": true,
+    "logging": "snapshot",
     "saved": true,
     "observer_can_run": true,
     "author_id": 1,
     "author_name": "John",
     "author_email": "john@example.com",
-    "packs": [
-      {
-        "created_at": "2021-01-19T17:08:31Z",
-        "updated_at": "2021-01-19T17:08:31Z",
-        "id": 14,
-        "name": "test_pack",
-        "description": "",
-        "platform": "",
-        "disabled": false
-      }
-    ]
+    "packs": []
```

Comment:
Re not returning `packs` the query belongs to: Are we ok with breaking the API here?

@zhumo

Reply:
@lucasmrod ahh sorry, I'll revert this change! I only intended to propose an update to how we _document_ packs in the response, no API change here — but since we're using this PR as reference for the API changes, this makes it confusing.

Reply:
OK, one more thing:

Here's a sample of a existing query response for a query that belongs to a global schedule:
```
{
  "query": {
    "created_at": "2023-06-28T23:07:56Z",
    "updated_at": "2023-06-28T23:38:39Z",
    "id": 4969,
    "name": "Get recently added or removed USB drives",
    "description": "Returns a record every time a usb device is plugged in or removed",
    "query": "SELECT action, DATETIME(time, 'unixepoch') AS datetime, vendor, mounts.path \nFROM disk_events \nLEFT JOIN mounts \n    ON mounts.device = disk_events.device\n;",
    "saved": true,
    "observer_can_run": false,
    "author_id": 155,
    "author_name": "Rachael Shaw",
    "author_email": "rachaelshaw@fleetdm.com",
    "packs": [
      {
        "created_at": "2021-09-17T05:28:54Z",
        "updated_at": "2021-09-17T05:28:54Z",
        "id": 1,
        "name": "Global",
        "description": "Global pack",
        "disabled": false,
        "type": "global",
        "labels": null,
        "label_ids": null,
        "hosts": null,
        "host_ids": null,
        "teams": null,
        "team_ids": null
      }
    ],
    "stats": {
      "system_time_p50": null,
      "system_time_p95": null,
      "user_time_p50": null,
      "user_time_p95": null,
      "total_executions": null
    }
  }
}
```

`packs` returned as of today here include: 2017 packs + all "schedules" because (to increase confusion around packs even more :) internally Fleet uses the term "packs" to group schedules in global or teams.

My guess is that we want to return 2017 packs only here now, not "schedules", because moving forward, queries won't be used by "schedules" anymore, but instead they belong to a single team (or global).

> but since we're using this PR as reference for the API changes

Yes. I believe the whole team is using this PR as source of truth.

Reply:
@lucasmrod yeah your assumption sounds right— should only be 2017 packs, and ideally we'd move away from using packs internally 👍